Sources/PackageDescription4/Target.swift

```diff
@@ -61,6 +73,20 @@ public final class Target {
     /// If a value is not provided, the directory will be set to "include".
     public var publicHeadersPath: String?
 
+    public var type: TargetType
```

Comment:
I don't think we want this to be public, same goes for the things below (I think)